sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyRequestAttributes.java

```diff
@@ -74,6 +76,12 @@
     @JsonProperty(value = "updated", access = JsonProperty.Access.WRITE_ONLY)
     private Long updated;
 
+    /**
+     * Indicates if the private key can be exported.
+     */
+    @JsonProperty(value = "exportable")
+    private Boolean exportable;
```

Comment:
Was this missing from previous versions?

Reply:
I added the `exportable` property to a few places based on the new swagger. I don't think it was supposed to be there before.